.github/workflows/publish-engines.yml

```diff
@@ -38,14 +38,23 @@ jobs:
           GITHUB_EVENT_CLIENT_PAYLOAD: ${{ toJson(github.event.client_payload) }}
 
       - name: Workflow dispatch to prisma/prisma for version update
-        if: ${{ steps.publish_script.outputs.new_prisma_version }}
+        # Only automatically trigger update for `latest`!
+        if: steps.publish_script.outputs.npm_dist_tag == 'latest' 
```

Comment:
`new_prisma_version` was only set in publish.ts if `npmDistTag == latest`.
This is now changed to _always_ be set, but the Npm dist tag itself is also made available as a variable.
Use that new variable instead to achieve the same result.